File: DSA/Krish_Mishra_590015053/Week_3/Day18_Question1.py
def selection_sort(arr):
    n = len(arr)
    for i in range(n):
        min_index = i
        for j in range(i+1, n):
            if arr[j] < arr[min_index]:
                min_index = j
        arr[i], arr[min_index] = arr[min_index], arr[i]
    return arr
user_input = input("Enter the numbers separated by spaces: ")
arr = list(map(int, user_input.split()))
sorted_arr = selection_sort(arr)
print("Sorted array:", sorted_arr)

File: DSA/Shreya_Goyal_590015491/Day_18_Question_1.c
#include <stdio.h>

int main() {
    int arr[5], i, j, mi, temp;

    printf("Enter 5 numbers:\n");
    for(i = 0; i < 5; i++) {
        scanf("%d", &arr[i]);
    }

    for(i = 0; i < 4; i++) {  
        mi = i;
        for(j = i + 1; j < 5; j++) {
            if(arr[j] < arr[mi]) {
                mi = j;
            }
        }

        temp = arr[i];
        arr[i] = arr[mi];
        arr[mi] = temp;
    }

    printf("Sorted array:\n");
    for(i = 0; i < 5; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}
